<launch>
    
    <node name='path_planner' pkg='leo_rover_localization' type='path_planner_service.py' args="$(find leo_rover_localization)/meshes/leo_map.npz"  />

    <node name='controller' pkg='leo_rover_localization' type='rover_controller_node.py' />
    
</launch>
